Makefile

```diff
@@ -125,16 +130,33 @@ oc-deploy-starter: ## Deploys basic prow infrastructure
 	@echo "Deploying prow infrastructure"
 	@oc apply -f cluster/starter.yaml
```

Comment:
Maybe we kick-out the "starter.yaml" entirely?

Reply:
ok, I'll remove it. You are right that we don't use it at all so there is no point in keeping it there. In addition, these images are not centos based...

Reply:
c629ccf